CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2
 
 ### 🐛 Bug Fixes
 
+* `[template/nextjs]` Remove `edge` next config plugin ([#1105](https://github.com/Sitecore/jss/pull/1105))
```

Comment:
We don't need a changelog entry (Next.js Edge compatibility wasn't introduced until this release)
```suggestion
```